src/Components/App/App.tsx

```diff
@@ -13,26 +11,24 @@ import "./library.jpg"
 const App: React.FC = () => {
   const [books, setBooks] = useState([])
 
-  const retrieveBooks = () => {
-    getBookByCategory('history')
-      .then((data: {works: []}) => setBooks(data.works))
-      .then(() => console.log(books))
```

Comment:
Excellent thinking in removing the console.log... the information shows up in the react dev tools